docs/reference/data-streams/tsds.asciidoc

```diff
@@ -0,0 +1,311 @@
+ifeval::["{release-state}"=="unreleased"]
+[[tsds]]
+== Time series data stream (TSDS)
+
+preview::[]
+
+A time series data stream (TSDS) models timestamped metrics data as one or
+more time series.
+
+// TODO: Replace XX% with actual percentage
+You can use a TSDS to store metrics data more efficiently. In our benchmarks,
+metrics data stored in a TSDS used XX% less disk space than a regular data
```

Comment:
I could swear we wrote this down somewhere in https://github.com/elastic/elasticsearch/issues/74660 or one of the many many linked PRs.

Reply:
Graham shared an all hands presentation with me that says ">50% storage reduction for metrics", so if you think that's okay to use, I'll put that in.

Reply:
50% seems like a lot. I've got that from synthetic_source, but not tsdb. Combining the two, totally. I'll rerun a test locally while I have my coffee this morning.

Reply:
|                                                    Store size |            |     5.69771     |     3.20667     |    -2.49105 |     GB |  -43.72% |


OK. 44%....

Reply:
Changed to:
>You can use a TSDS to store metrics data more efficiently. In our benchmarks,
>metrics data stored in a TSDS used 44% less disk space than a regular data
>stream.